pkg/windows/windows.go

```diff
@@ -233,9 +229,6 @@ type Windows interface {
 
 // windows implements the Windows interface
 type windows struct {
-	// workerIgnitionEndpoint is the Machine Config Server(MCS) endpoint from which we can download the
-	// the OpenShift worker ignition file.
-	workerIgnitionEndpoint string
```

Comment:
Consider mentioning in the commit message why `workerIgnitionEndpoint` is no longer relevant.